pkg/kev/envs.go

```diff
@@ -164,6 +163,14 @@ func (e *Environment) GetVolume(name string) (VolumeConfig, error) {
 	return e.override.getVolume(name)
 }
 
+// GetVolume retrieves a specific volume by name from the environment's override volumes.
```

Comment:
I guess this comment isn't for this method?

Reply:
Fixed!